PrestaShop/PrestaShop
# [-] BO: Fix CLDR & Globalize javascripts

## Description

Fix the missing CLDR datas needed by Globalize on every pages using Globalize.
## Steps to Test this Fix

Multiple methods:
- Check if there is no error on:
  - Dashboard
  - Products listing
  - Product page
- Create an order from the backoffice
## Forge Ticket (optional)
- BOOM-286
- BOOM-284
- BOOM-280
- BOOM-279

(_and maybe many more_)
